modules/sensor/src/rgb-depth/realsense/vpRealSense2.cpp

```diff
@@ -423,13 +631,14 @@ vpCameraParameters vpRealSense2::getCameraParameters(const rs2_stream &stream,
 /*!
    Get intrinsic parameters corresponding to the stream. This function has to
    be called after open().
-   \param stream : stream for which the camera intrinsic parameters are returned.
+   \param stream : Stream for which the camera intrinsic parameters are returned.
+   \param index  : Index of the stream.
```

Comment:
Description of `index` to complete here, and maybe each time this parameter is used in a method